tox.ini

```diff
@@ -14,7 +14,7 @@ isolated_build = True
 
 [testenv]
 deps = pytest
-commands = pytest tests
+commands = pytest {posargs}
```

Comment:
Making this change allows me to use `tox` to run specific tests/paths like so:

    tox -e py36 tests/test_check_raises.py::test_msg_kwarg_with_raises_context_manager

If you have another way of doing this, I'll revert this change.